pytensor/link/numba/dispatch/slinalg.py

```diff
@@ -126,17 +126,17 @@ def impl(A, B, trans, lower, unit_diagonal, b_ndim, overwrite_b):
 
         B_is_1d = B.ndim == 1
 
-        if overwrite_b:
-            B_copy = B
-        else:
-            if B_is_1d:
-                # _copy_to_fortran_order does nothing with vectors
-                B_copy = np.copy(B)
-            else:
-                B_copy = _copy_to_fortran_order(B)
+        A_copy = _copy_to_fortran_order(A)
```

Comment:
Why do we need a copy? Is it enough for A to be fortran? Or do these routines inplace on A by default?

Reply:
Isn't there a numba `asfortranarray`?

Reply:
Actually on second though we can just call `np.asfortranarray(A)` and it will handle copying or not automatically. That's what we're doing for B anyway.